packages/core/src/application_ref.ts

```diff
@@ -682,20 +682,51 @@ export class ApplicationRef {
   }
 
   /**
-   * Bootstrap a new component at the root level of the application.
+   * Bootstrap a component onto the element identified by its selector or, optionally, to a
+   specified
+   * element
    *
    * @usageNotes
    * ### Bootstrap process
    *
-   * When bootstrapping a new root component into an application, Angular mounts the
-   * specified application component onto DOM elements identified by the componentType's
-   * selector and kicks off automatic change detection to finish initializing the component.
+   * When bootstrapping an application with a component, Angular mounts the specified application
+   * component onto a DOM element, and kicks off automatic change detection to finish initializing
+   * the component.
+   *
+   * If not provided in the `rootSelectorOrNode` argument, the selector of the provided
+   * `componentOrFactory` will be used to match the DOM elements.
+   *
+   * Please note that if multiple elements match the selector, the first matched element will be
+   * used as a host element.
```

Comment:
I'd propose to rephrase this a bit:
```suggestion
   * When bootstrapping a component, Angular mounts it onto a target DOM element and kicks off automatic
   * change detection. The target DOM element can be provided using the `rootSelectorOrNode` argument. If 
   * the target DOM element is not provided, Angular tries to find one on a page using a `selector` of the
   * component that is being bootstrapped (first matched element is used).
```